hnn_core/batch_simulate.py

```diff
@@ -40,31 +41,15 @@ def __init__(self, set_params, net=jones_2009_model(), tstop=170,
             The network model to use for simulations. Must be an instance of
             jones_2009_model, law_2021_model, or calcium_model.
             Default is jones_2009_model().
+        net_json : str, optional
```

Comment:
Since a `Network` is already being passes in during the init, it might be easier to have users create the network with a file outside of `BatchSimulate`, and then pass it to the init.

The comment I made was mainly referring to whether or not it is worth exporting these `net_json` files after `batch_simulate.run()` is called. There's a lot of redundant information in those files so it's definitely not worth saving 100,000 copies of the same network that changes by a handful of parameters...

However, there should be some metadata that helps people understand what the exported parameter values mean (it's not clear to me how to best achieve that right now)